test/check-application

```diff
@@ -1084,8 +1082,9 @@ class TestApplication(testlib.MachineCase):
             self.assertIn(full[1], ["GB", "MB"])
 
             # Test that the value is updated dynamically
-            self.execute(auth, "podman exec -i swamped-crate sh -c 'dd bs=1024 count=1000000 < /dev/urandom > /dev/null'")
+            self.execute(auth, "podman exec -di swamped-crate sh -c 'dd bs=1024 < /dev/urandom > /dev/null'")
```

Comment:
Hmm, that means the `podman exec` command itself will stay around for that long? Does that block the `restart`, or is that fast? I'd feel a bit easier if this would use `&` instead of `-d`, and `pkill dd` instead of `podman restart`.

There are lots of flakes, perhaps that would help a bit?

Reply:
There are a ton of unrelated flakes, I don't see how they relate to `podman exec`. But maybe a kill is better as a restart is slow.